src/definitions/enumType.ts

```diff
@@ -1,6 +1,10 @@
 import { NexusTypes, withNexusSymbol } from "./_types";
 import { assertValidName } from "graphql";
 
+type TypeScriptEnumLike = {
```

Comment:
I'm not 100% happy with this but I couldn't find a better way to allow a TypeScript enum definition to fit into the typing, open to better suggestions

Reply:
This is the best solution I can find as well, so I think it's probably alright.